frontend/src/main/java/puf/frisbee/frontend/view/BackgroundImageView.java

```diff
@@ -0,0 +1,30 @@
+package puf.frisbee.frontend.view;
+
+import javafx.fxml.FXML;
+import javafx.scene.image.ImageView;
+import puf.frisbee.frontend.core.ViewHandler;
+import puf.frisbee.frontend.viewmodel.GameViewModel;
+import puf.frisbee.frontend.viewmodel.WaitingViewModel;
+
+public class BackgroundImageView {
+    @FXML
+    private ImageView backgroundImage;
+
+    private WaitingViewModel waitingViewModel;
+    private GameViewModel gameViewModel;
+    private ViewHandler viewHandler;
+
+    public void init(WaitingViewModel waitingViewModel, ViewHandler viewHandler) {
+        this.waitingViewModel = waitingViewModel;
+        this.viewHandler = viewHandler;
```

Comment:
Für was brauchen wir den viewHandler in dieser View? Ich glaube, der kann überall raus 🤔 ?

Reply:
ViewHandler ist raus.